src/components/EnvoyFleets/EnvoyFleetsList/EnvoyFleetsList.tsx

```diff
@@ -44,32 +44,30 @@ const EnvoyFleetsList: React.FC = () => {
 
   return (
     <ContentWrapper>
-      <ListTableTitleContainer>
-        <ListTableTitleLabel>Envoy Fleets</ListTableTitleLabel>
-
-        <S.TitleFiltersContainer>
-          {loading ? (
-            <Skeleton.Button />
-          ) : error ? null : (
-            <S.Select
-              allowClear
-              placeholder="Select a namespace"
-              value={selectedNamespace}
-              showSearch
-              onClear={onNamespaceSelectionClearHandler}
-              onSelect={(value: any) => {
-                onNamespaceSelectHandler(value);
-              }}
-            >
-              {envoyFleetsNamespaces.map(namespace => (
-                <Option key={namespace} value={namespace}>
-                  {namespace}
-                </Option>
-              ))}
-            </S.Select>
-          )}
-        </S.TitleFiltersContainer>
-      </ListTableTitleContainer>
+      <ListTableTitleLabel>Envoy Fleets</ListTableTitleLabel>
+
+      <S.TitleFiltersContainer>
+        {loading ? (
```

Comment:
You can wrap your S.Select with Skeleton like shown on the screenshot. Try it. If you like it - use it. If no - leave it as is
<img width="316" alt="image" src="https://user-images.githubusercontent.com/25991946/164425872-b9e5c9c0-d48d-4811-b02f-519b86755895.png">